src/main/java/org/jabref/model/entry/AuthorList.java

```diff
@@ -11,6 +11,8 @@
 import java.util.WeakHashMap;
 import java.util.stream.Collectors;
 
+import org.jabref.model.strings.LatexToUnicodeAdapter;
+
```

Comment:
The changes in AuthorList are intended to create a similar behaviour to the already existing `getAsNatBib`. An alternative would be to create the cache as a weakhashmap in `MainTableNameFormatter` (and move the functionality from `model` to `gui` if that is appropriate)

Reply:
I think it's reasonable to have it in the AuthorList class.

Reply:
Good! Then I will start writing test cases for it and see if I can find something that doesn't work :)
Thank you for taking a look at it!